crypto/aes/asm/aesni-x86_64.pl

```diff
@@ -4296,8 +4296,8 @@ sub aesni_generate8 {
 
 # This is based on submission by
 #
-#	Huang Ying <ying.huang@intel.com>
-#	Vinodh Gopal <vinodh.gopal@intel.com>
+#	Huang Ying
+#	Vinodh Gopal
 #	Kahraman Akdemir
```

Comment:
"based on submission from Intel by". [Though Intel people might have special opinion about whether or not it's just "Intel" or "Intel Corp."]

Reply:
fixed